controller/lib/core/comms.cpp

```diff
@@ -15,23 +15,24 @@
 // beginning of the buffer.
 static uint8_t tx_buffer[ControllerStatus_size];
 // Index of the next byte to transmit.
-static uint8_t tx_idx = 0;
+static uint16_t tx_idx = 0;
 // Number of bytes remaining to transmit. tx_idx + tx_bytes_remaining equals
 // the size of the serialized ControllerStatus proto.
 static uint16_t tx_bytes_remaining = 0;
 
 // Time when we started sending the last ControllerStatus.
 // TODO: Change this to std::optional<Time> once that's available; then we
-// don't need this "clever" -1 initialization.
-static Time last_tx = millisSinceStartup(-1);
+// don't need this "clever" initialization.
+constexpr Time kInvalidTime = millisSinceStartup(0xFFFFFFFFFFFFFFFFUL);
```

Comment:
Prefer `uint64_t{-1}` or something like that; otherwise, how do I know that you've typed the correct number of `F`s?